controllers/operands/kubevirt_test.go

```diff
@@ -3067,7 +3067,7 @@ Version: 1.2.3`)
 					Expect(err).ToNot(HaveOccurred())
 					Expect(kv.Spec.Infra).To(Not(BeNil()))
 					Expect(kv.Spec.Infra.Replicas).To(Not(BeNil()))
-					Expect(*kv.Spec.Infra.Replicas).To(Equal(uint8(1)))
+					Expect(*kv.Spec.Infra.Replicas).To(HaveValue(Equal(uint8(1))))
```

Comment:
`HaveValue` is working on the pointer - not the value.
```suggestion
					Expect(kv.Spec.Infra.Replicas).To(HaveValue(Equal(uint8(1))))
```